docs/running-project.md

```diff
@@ -24,6 +24,8 @@ You need to initialize some data before pushing uData to it's full potential:
 $ udata init
 # Fetch and load licenses
 $ udata licenses https://www.data.gouv.fr/api/1/datasets/licenses
+# Build the search index
+$ udata search init
```

Comment:
There is two choices:
- either you loaded the fixture during the init and so the init should perform the search indexation
- either you didn't and so you don't have data to index here.

So, proposal: include the search indexation after the fixture loading in the `udata init` command